pkg/model/volumes.go

```diff
@@ -141,7 +141,6 @@ func (dev *Dev) getSourceSubPath(path string) string {
 			}
 			if filepath.IsAbs(path) {
 				oktetoLog.Info("could not retrieve subpath")
-				path = filepath.Base(path)
```

Comment:
This is the only thing that scares me a bit. We have faced several issues in the past with paths and things like that. I know this is not being used but it can be just because of a bug. @AgustinRamiroDiaz Can you check when this started to be useless to see if it was just that we forgot to removed or a potential bug?

Reply:
Talked offline with @AgustinRamiroDiaz . It seems that this line was introduced some weeks ago with a possible fix and the first release where it was included was the `2.5.0`, so not sure if it is a real bug (if we delete this line, the if doesn't do anything and not sure it is correct). So we'll create a ticket to take a look at that carefully.

Agustin, can you create it and post it here?

Reply:
Done! https://github.com/okteto/okteto/issues/2967

Reply:
We can safely remove this line, it was added because of a bug and after that, we fixed another one and we left that line but we should have cleaned that before

Reply:
but @jLopezbarb that line was introduced in the change just previous to this removal. Without that line, the if doesn't make sense. Are we sure it wasn't done on purpose?

Reply:
This can be removed. `path` is mutated inside the `if` but never used after that. It is used in the `else` which doesn't affect. We can probably also reverse the `if`

Reply:
I agree with @ifbyol, this could be a potential bug where we forget to use a variable. That's why I wanted one more look from @jLopezbarb, who knows which behavior we need here

Reply:
Since we created a new issue for that, can we revert this change and work on it in another PR?